src/util/DbrHelper.sol

```diff
@@ -325,4 +334,47 @@ contract DbrHelper is Ownable, ReentrancyGuard {
             revert SellPercentageTooHigh();
         if (repay.percentage > DENOMINATOR) revert RepayPercentageTooHigh();
     }
+
+
+       /**
+     * @notice Set a new pending gov. The new pending gov then has to call `acceptGov`.
+     * @dev Can only be called by the gov.
+     * @param _pendingGov address of the new pending gov
+     */
+    function setPendingGov(address _pendingGov) external onlyGov {
+        emit NewPendingGov(pendingGov, _pendingGov);
+        pendingGov = _pendingGov;
+    }
+
+    /**
+     * @notice Accept the new pending gov.
+     * @dev Can only be called by the pending gov.
+     */
+    function acceptGov() external {
+        require(msg.sender == pendingGov, "Only pending gov");
+        emit NewGov(gov, pendingGov);
+        gov = pendingGov;
+        pendingGov = address(0);
+    }
+
+    /**
+    @notice Sets a new curve pool
+    @dev Can only be called by the gov
+    @param _pool Address of the new curve pool
+    @param _dolaIndex Index of DOLA in the new curve pool
+    @param _dbrIndex Index of DBR in the new curve pool
+    @param _invIndex Index of INV in the new curve pool
+    */
+    function setCurvePool(address _pool, uint256 _dolaIndex, uint256 _dbrIndex, uint256 _invIndex) external onlyGov {
```

Comment:
Either check that indexes are correct or use similar method for setting indexes as used in constructor.